.gitignore

```diff
@@ -17,22 +16,16 @@
 #
 # You should have received a copy of the GNU General Public License version
 # 2 along with this work; if not, see <http://www.gnu.org/licenses/>.
-#
 # ===========================================================================
 #
-# exclude build directories
-/build
-/.build
 # exclude all source directories
 /omr
 /openj9
 /openssl
 # exclude optional eclipse project file
 /.project
```

Comment:
This duplicated entry can be removed like https://github.com/ibmruntimes/openj9-openjdk-jdk/pull/599.

Reply:
Refer https://github.com/ibmruntimes/openj9-openjdk-jdk17/pull/211